src/platform/user/profile/utilities/index.js

```diff
@@ -193,7 +193,7 @@ export const getValidationMessageKey = (
     !containsBadUnitNumber
   ) {
     return validationKey
-      ? ADDRESS_VALIDATION_TYPES.SHOW_SUGGESTIONS_OVERRIDE
+      ? ADDRESS_VALIDATION_TYPES.SHOW_SUGGESTIONS_NO_CONFIRMED_OVERRIDE
```

Comment:
would you be so kind as to create a ticket to add unit tests for this function?

Reply:
@erikphansen we can add that to the e2e test coverage ticket